knowledge-base/how-to-view-web-session-summary.md

```diff
@@ -0,0 +1,101 @@
+---
+title: How to view Web Session Summary
+description: An article on how to capture and inspect web traffic using Fiddler Everywhere
+type: how-to
+slug: how-to-capture-and-inspect-web-traffic
+tags: capture web traffic, inspect web traffic, web traffic, https, web application
+publish: true
+res_type: kb
+---
+
+## View Sessions Summary
+
+Each __Session__ captured by Fiddler Everywhere has multiple attributes ordered in columns. Use these to identify HTTP related issues, to track and debug your site performance, and to leave custom comments.
+
+## (ID)
```

Comment:
Why is ID in parentheses? Also, it would be nice to note that somehow that you are talking about the # field. Perhaps you can use: **ID (#)** or something similar